### PR TITLE
Improve/multiline labels

### PR DESCRIPTION
**Pull request by zenangst**
Adds support for multiline line descriptions

screen shot 2015-04-21 at 13 29 05

https://github.com/hyperoslo/Form/issues/143

### CODE REVIEW

File: Source/Cells/Popover/FORMFieldValuesTableViewController.m [hunk 1]
```diff
@@ -57,11 +66,14 @@ - (UIView *)tableView:(UITableView *)tableView viewForHeaderInSection:(NSInteger
...
 - (CGFloat)tableView:(UITableView *)tableView heightForHeaderInSection:(NSInteger)section {
-    if (self.customHeight) {
-        return self.customHeight;
-    } else {
-        return (self.field.info) ? FORMFieldValuesHeaderHeight : FORMFieldValuesCellHeight;
-    }
+        if (self.field.info) {
+            [self.headerView setField:self.field];
+            return [self.headerView labelHeight];
+        } else if (self.customHeight > 0.0f) {
```
Comment (3lvis) [inline Source/Cells/Popover/FORMFieldValuesTableViewController.m]: Could you drop the multiple returns?

File: Source/Cells/Popover/FORMFieldValuesTableViewHeader.h [hunk 2]
```diff
@@ -11,6 +12,8 @@ static NSString * const FORMFieldValuesTableViewHeaderIdentifier = @"FORMFieldVa
 
 @property (nonatomic, weak) FORMField *field;
 
+- (CGFloat)labelHeight;
```
Comment (3lvis) [inline Source/Cells/Popover/FORMFieldValuesTableViewHeader.h]: Can you add tests for this?

File: Source/Cells/Popover/FORMFieldValuesTableViewController.m (hunk 1 above)
Comment (zenangst) [inline Source/Cells/Popover/FORMFieldValuesTableViewController.m]: Fixed

File: Tests/Tests/FORMFieldValuesTableViewHeaderTests.swift [hunk 3]
```diff
@@ -0,0 +1,31 @@
+//
+//  FORMFieldValuesTableViewHeaderTests.swift
+//  Tests
+//
+//  Created by Christoffer Winterkvist on 4/21/15.
```
Comment (3lvis) [inline Tests/Tests/FORMFieldValuesTableViewHeaderTests.swift]: Can you remove this?

File: Tests/Tests/Tests-Bridging-Header.h [hunk 4]
```diff
@@ -0,0 +1,6 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
```
Comment (3lvis) [inline Tests/Tests/Tests-Bridging-Header.h]: Can you remove this?

Comment (zenangst) [inline Tests/Tests/Tests-Bridging-Header.h]: Removed

File: Tests/Tests/FORMFieldValuesTableViewHeaderTests.swift (hunk 3 above)
Comment (zenangst) [inline Tests/Tests/FORMFieldValuesTableViewHeaderTests.swift]: Removed

File: Source/Cells/Popover/FORMFieldValuesTableViewHeader.h (hunk 2 above)
Comment (zenangst) [inline Source/Cells/Popover/FORMFieldValuesTableViewHeader.h]: Added a test here https://github.com/hyperoslo/Form/blob/improve/multiline-labels/Tests/Tests/FORMFieldValuesTableViewHeaderTests.swift

**Comment by 3lvis** [comment]
:tada:

**Pull request merged**